### PR TITLE
fix: no actions in w3c actions case

### PR DESCRIPTION
**Pull request by KazuCocoa**
Let's update existing e2e for send_keys with w3c action in selenium client way.
For now, this test should fail without https://github.com/appium/WebDriverAgent/pull/919.

### CODE REVIEW

**Comment by KazuCocoa** [comment]
oh...

> 9% of tests failed. This is (probably) fine

So this test failed actually but passed. Let me make the threshold strict a bit more

File: test/functional/basic/element-e2e-specs.js [hunk 1]
```diff
@@ -320,6 +320,16 @@ describe('XCUITestDriver - elements -', function () {
...
 
           const actions = [
+            // Selenium clients generate below code for `driver.action.send_keys('a').perform`.
+            {
+              type: 'pointer',
+              id: 'touch',
+              actions: [
+                {type: 'pause', duration: 0},
+                {type: 'pause', duration: 0},
+                {type: 'pause', duration: 0}
```
Comment (kkb912002) [inline test/functional/basic/element-e2e-specs.js:330]: should be 6 times.

File: .github/workflows/functional-test.yml [hunk 2]
```diff
@@ -118,7 +118,7 @@ jobs:
           failures=$(cat "$RESULTS_XML" | xq --xpath '//testsuite/@failures')
           threshold=$(( (failures + errors) * 100 / (tests - skipped) ))
           cat "$RESULTS_XML"
-          if [[ $threshold -gt 10 ]]; then
+          if [[ $threshold -gt 7 ]]; then
```
Comment (mykola-mokhnach) [inline .github/workflows/functional-test.yml:121]: not sure we are ready to do this yet

**Comment by KazuCocoa** [comment]
hm... 
```
> appium-xcuitest-driver@7.24.0 lint
> eslint .

sh: 1: eslint: not found
Error: Process completed with exit code 12
```
https://github.com/appium/appium-xcuitest-driver/actions/runs/10002129669/job/27646928703?pr=2432

**Comment by mykola-mokhnach** [comment]
> hm...
> 
> ```
> > appium-xcuitest-driver@7.24.0 lint
> > eslint .
> 
> sh: 1: eslint: not found
> Error: Process completed with exit code 12
> ```
> 
> https://github.com/appium/appium-xcuitest-driver/actions/runs/10002129669/job/27646928703?pr=2432
> 
> It looks like this is CI side issue. I ran https://github.com/appium/appium-xcuitest-driver/actions/runs/9890897067 today, then the same result came while the same lint passed yesterday.

It's related to https://github.com/npm/cli/issues/7667

File: .github/workflows/unit-test.yml [hunk 3]
```diff
@@ -27,6 +27,7 @@ jobs:
     needs:
     - prepare_matrix
     strategy:
+      fail-fast: false
```
Comment (KazuCocoa) [inline .github/workflows/unit-test.yml:30]: unit test is pretty fast, so `fail-fast: false` should be good

**Review by mykola-mokhnach — APPROVED** [review]

**Pull request merged**